aws/s3/main.tf

```diff
@@ -0,0 +1,69 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.28.0"
+    }
+  }
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = "us-east-2"
+}
+
+// Create a generic bucket
+resource "aws_s3_bucket" "demo_bucket" {
+  bucket = "my-demo-scan-bucket"
+}
+
+resource "aws_s3_object" "github_object" {
```

Comment:
It would be nice to add a private key for both Parameter store and S3, we should be able to generate it instead of hardcoding

Reply:
Definitely seems reasonable. lemme try it out.

Reply:
Added to both and verified scanning was picking it up